### DISCUSSION TITLE
How do I set a route as an Index if it requires a parameter 🙏🏽

### DISCUSSION THREAD

**Discussion by deyemiobaa** [remix-run/react-router#9105] [Q&A]
I was following the [quick start guide](https://reactrouter.com/docs/en/v6/getting-started/overview#index-routes) on react-router, and I came across this code sample that shows how to direct a route to a nested route by default using index.

Now my question is if the path you're trying to switch to requires a parameter :someID how would I set it as the index with a default value? e.g /23

```jsx
function App() {
  return (
    <Routes>
      <Route path="/" element={<Layout />}>
        <Route index element={<Activity />} />
        <Route path="invoices" element={<Invoices />} />
        <Route path="activity" element={<Activity />} />
        <Route path=":someID" element={<SomePath />} />
      </Route>
    </Routes>
  );
}
```

**Comment by slk333** [comment]
That's the issue with index routes, they are kinda useless in v6.
The workaround seems to do a redirect from the index to a specific subroute, such as
```js
<Route index element={<Navigate to="123" replace={true} />} />
```
